lib/chef/knife/vsphere_vm_clone.rb

```diff
@@ -173,67 +172,6 @@ class Chef::Knife::VsphereVmClone < Chef::Knife::BaseVsphereCommand
          proc: proc { |key| Chef::Config[:knife][:bootstrap_protocol] = key },
          default: nil
 
-  option :ssh_user,
-         short: "-x USERNAME",
-         long: "--ssh-user USERNAME",
-         description: "The ssh username",
-         default: "root"
-
-  option :ssh_password,
-         short: "-P PASSWORD",
-         long: "--ssh-password PASSWORD",
-         description: "The ssh password"
-
-  option :ssh_port,
-         short: "-p PORT",
-         long: "--ssh-port PORT",
-         description: "The ssh port",
-         default: "22"
-
-  option :identity_file,
-         short: "-i IDENTITY_FILE",
-         long: "--identity-file IDENTITY_FILE",
-         description: "The SSH identity file used for authentication"
-
-  option :chef_node_name,
-         short: "-N NAME",
-         long: "--node-name NAME",
-         description: "The Chef node name for your new node"
-
-  option :prerelease,
-         long: "--prerelease",
-         description: "Install the pre-release chef gems",
-         boolean: false
-
-  option :bootstrap_version,
-         long: "--bootstrap-version VERSION",
-         description: "The version of Chef to install",
-         proc: proc { |v| Chef::Config[:knife][:bootstrap_version] = v }
-
-  option :bootstrap_proxy,
-         long: "--bootstrap-proxy PROXY_URL",
-         description: "The proxy server for the node being bootstrapped",
-         proc: proc { |p| Chef::Config[:knife][:bootstrap_proxy] = p }
-
-  option :bootstrap_vault_file,
-         long: "--bootstrap-vault-file VAULT_FILE",
-         description: "A JSON file with a list of vault(s) and item(s) to be updated"
-
-  option :bootstrap_vault_json,
-         long: "--bootstrap-vault-json VAULT_JSON",
-         description: "A JSON string with the vault(s) and item(s) to be updated"
-
-  option :bootstrap_vault_item,
-         long: "--bootstrap-vault-item VAULT_ITEM",
-         description: 'A single vault and item to update as "vault:item"',
-         proc: proc { |i|
-           (vault, item) = i.split(/:/)
-           Chef::Config[:knife][:bootstrap_vault_item] ||= {}
-           Chef::Config[:knife][:bootstrap_vault_item][vault] ||= []
-           Chef::Config[:knife][:bootstrap_vault_item][vault].push(item)
-           Chef::Config[:knife][:bootstrap_vault_item]
-         }
-
   option :distro,
```

Comment:
Could be removed because chef and other knife plugin no longer support `distro` .

Reply:
removed `distro` option.